substrate/frame/revive/src/tests/stipends.rs

```diff
@@ -0,0 +1,302 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::{
+	test_utils::{builder::Contract, ALICE, BOB_ADDR},
+	tests::{builder, ExtBuilder, RuntimeEvent, Test},
+	BalanceWithDust, Code, Config, Pallet, System,
+};
+use alloy_core::sol_types::{SolCall, SolEvent};
+use frame_support::traits::fungible::Mutate;
+use pallet_revive_fixtures::{
+	compile_module_with_type, ComplexReceiver, FixtureType, SimpleReceiver, StipendSender,
+};
+use test_case::test_case;
+
+pub use sp_core::H160;
```

Comment:
```suggestion
use test_case::test_case;
use sp_core::H160;
```

Reply:
Changed in https://github.com/paritytech/polkadot-sdk/pull/10166/commits/116ba4f6d7596a70c912585d02f44647a57f5d66.